themes/qgrid/source/css/nav.scss

```diff
@@ -16,6 +16,10 @@
     height: 69px;
     border-bottom: 1px solid white;
     display: none;
+
+    .search {
+      width: calc(100% - 34px);
```

Comment:
can we apply margin/padding instead of calc?

Reply:
I have to stretch this input